### ISSUE TITLE
Propogate "--key-password" websockify option to launch.sh

### ISSUE THREAD

**Issue by EgorDuplensky** [novnc/noVNC#1490]
**Is your feature request related to a problem? Please describe.**
Currently it is not possible to use encrypted cert key when running websockify via utils/launch.sh

**Describe the solution you'd like**
Expose "--key-password" to utils/launch.sh

**Describe alternatives you've considered**
N/A

**Additional context**
I can prepare a PR if maintainers are OK with the proposal

**Comment by CendioOssman** [comment]
We propagate various other things, so I don't see why we can't add this as well.

**Comment by EgorDuplensky** [comment]
Btw, It seems that websockify in general is way ahead in terms of command arguments compared to launch.sh. 
I see some checks are performed for some options before passing to websockify.
Maybe it is better to just forward the arguments that launch.sh is not aware about to websockify?

**Comment by CendioOssman** [comment]
Yeah, `launch.sh` was just really a quick hack to make it easier to test and develop noVNC. So soem rethinking is probably needed to avoid all that duplication. Please send us a suggestion as a PR if you have the time.
